### PR TITLE
Cleanup StatefulSet PVC after deleting polaris.

### LINKED ISSUE #946: Fix how PV labels are set in Polaris StatefulSet
The labels of PVs created by statefulset are overwritten in k8s < 1.15 and set to the same values as the labelSelector. Therefore, the label selector of Eventstore and Consul need to be updated to ensure compatibility with older Kubernetes and facilitate cleanup operations in synopsysctl when a polaris instance is deleted.

### PR DESCRIPTION
**Pull request by jdartigalongue**
Closes #946

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/954#" title="Author self-approved">jdartigalongue</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [jdartigalongue]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_delete.go [hunk 1]
```diff
@@ -190,6 +194,15 @@ var deletePolarisCmd = &cobra.Command{
 			}
 		}
 
+		// Delete StatefulSet PVCs
+		if list, err := kubeClient.CoreV1().PersistentVolumeClaims(namespace).List(metav1.ListOptions{LabelSelector: fmt.Sprintf("environment=%s", p.Namespace)}); err == nil {
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_delete.go]: Are you not using pkg/common methods? :-)

Comment (msenmurugan) [reply pkg/synopsysctl/cmd_delete.go]: List PVC's, Get PVC's method

File: pkg/synopsysctl/cmd_delete.go [hunk 2]
```diff
@@ -190,6 +194,15 @@ var deletePolarisCmd = &cobra.Command{
...
+			for _, v := range list.Items {
+				if err := kubeClient.CoreV1().PersistentVolumeClaims(namespace).Delete(v.Name, &metav1.DeleteOptions{}); err != nil {
```
Comment (msenmurugan) [inline pkg/synopsysctl/cmd_delete.go]: Whether we need to support this use case?

Comment (msenmurugan) [reply pkg/synopsysctl/cmd_delete.go]: If so, can we provide some warning if the reclaim policy is `delete`?

Comment (jdartigalongue) [reply pkg/synopsysctl/cmd_delete.go]: As discussed, there is now a warning and the user is asked whether the PVC should be deleted or not.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/954#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/954#pullrequestreview-298972215" title="LGTM">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [jdartigalongue,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**